### ISSUE TITLE
Release fo HK2 2.6

### ISSUE THREAD

**Issue by smillidge** [eclipse-ee4j/glassfish-hk2#456]
Release 2.6
AFAIK we need these CQs

- [x] Upgrade to ASM 7.1 (Already has an approved CQ) See https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19521

**Comment by Cousjava** [comment]
@smillidge CQ already done - https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19522

**Comment by smillidge** [comment]
yeah I did a piggyback CQ

**Comment by buko** [comment]
Any news on this?

**Comment by smillidge** [comment]
CQs have been approved. When I have time I will kick off an RC build and a release review.

**Comment by smillidge** [comment]
We are going to have to kick off a new CQ due to the Guava upgrade

**Comment by romain-grecourt** [comment]
@smillidge Let me know if you need any help to move this forward.

**Comment by smillidge** [comment]
All help is appreciated. Just back from holiday but I think we just need a CQ for the Guava upgrade and then follow the release process. It's the release process I'm unfamiliar on.

**Comment by romain-grecourt** [comment]
OK, let me know when the CQ is through ; I'll take care of the Maven release.

**Comment by batsatt** [comment]
@smillidge Any update on the CQ?

**Comment by m0mus** [comment]
@smillidge If it's a minor release you don't need a release review. Let me know what version of Guava and full maven coordinates are used and I can help with CQ.

**Comment by smillidge** [comment]
Piggyback CQ created https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20480 once this is done we can kick off the release. Thanks @m0mus for reminding me we don't need a release review.

**Comment by m0mus** [comment]
@smillidge CQ https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20480 is approved and we can proceed with the release. Let me know if you need help.

**Comment by romain-grecourt** [comment]
@m0mus Are we okay to kick-off the Maven release ? If so I can take it from here.

**Comment by smillidge** [comment]
Go for it. There is already a release record on the project.

**Comment by smillidge** [comment]
@romain-grecourt can you document the process so others can kick it off in the future?

**Comment by romain-grecourt** [comment]
The process is still clunky and needs polishing, but in short this is done with 2 jobs:
- https://jenkins.eclipse.org/glassfish/job/release/
- https://jenkins.eclipse.org/glassfish/job/nexus-release/

Some time ago I spent some time to polish this up, but this work is not finished.
The idea is to gather all the materials (including release instructions) in this repository: https://github.com/eclipse-ee4j/glassfish-infra

**Comment by romain-grecourt** [comment]
Closing this issue since version 2.6.0 is now available on Maven Central and the version was updated in the master branch to 2.6.1-SNAPSHOT.
